src/kakarot/instructions/logging_operations.cairo

```diff
@@ -60,12 +60,8 @@ namespace LoggingOperations {
             self=ctx.memory, element_len=actual_size, element=data, offset=actual_offset
         );
 
-        let ctx = ExecutionContext.push_event_to_emit(
-            self=ctx,
-            event_keys_len=topics_len * 2,
-            event_keys=popped + 4,
-            event_data_len=actual_size,
-            event_data=data,
+        let ctx = ExecutionContext.push_event(
+            self=ctx, topics_len=topics_len, topics=popped + 4, data_len=actual_size, data=data
```

Comment:
looking back, and i see we've named quite a few variables actual_.... in eftests and kakarot
i think it's confusing, what does it even mean